addons/portal/views/portal_templates.xml

```diff
@@ -228,13 +228,6 @@
                     <div t-if="portal_service_category_enable" class="o_portal_category row g-2 mt-3" id="portal_service_category"/>
                     <div t-if="portal_vendor_category_enable" class="o_portal_category row g-2 mt-3" id="portal_vendor_category"/>
                     <div class="o_portal_category row g-2 mt-3" id="portal_common_category">
-                        <t t-call="portal.portal_docs_entry">
```

Comment:
maybe we could just do

```suggestion
<t t-call="portal.portal_docs_entry" t-if="False">
```
to be less XPATH risky (because this is really a 0 added value fix, so if it cause an issue it would be sad)